src/app/manage/manage.component.html

```diff
@@ -36,6 +36,7 @@
                 {{settings.lang.smsTextHelpTitle}}
                 <ul>
                     <li>!מתנדב! - {{settings.lang.replacedByVolunteerName}}</li>
+                    <li>!תורם! - {{settings.lang.replacedByHelperName}}</li>
```

Comment:
This is wrong - should be replaced by FamilyName -  and also !תורם! should be !משפחה!  or with some specificity to mlt